### PR TITLE
Create q4_spectaprod

### PR DESCRIPTION
**Pull request by spectaprod**
QA proposal

### CODE REVIEW

**Review by riongull — CHANGES_REQUESTED** [review]
Please see in-line comments.  Let's get on the phone or Discord to discuss monthly budget in context of our overall budget request.

File: internal/q4_spectaprod [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+### Project Updates & Budget Spending
+
+In Q4 2022 I funded the following Bounties:
+
+* [Dash Ecosystem Infographic](https://trello.com/c/L306RZDn) - This was an inherited bounty, and is nearing completion. This is a graphical representation of the entire Dash ecosystem for quick reference. Link to the current state is in below.
+* [Interactive Dash Ecosystem Webpage](https://trello.com/c/djmlrMDC) - This was an inherited bounty, and is nearing completion. This is closely related to the infographic (above). It is the "presentation layer" presenting the infographic for easy display and navigation in a web browser across mulitple languages. You can see the infographic, available interactions, and hosting environment [here] (https://media.dash.org/wp-content/uploads/Dash-Ecosystem-English-4K-2022-05-11.jpg)
```
[the remark this answers is not in the document]
Comment (spectaprod) [reply internal/q4_spectaprod]: as far as the 2 I have (you have the 3rd), the difference is the infographic is the graphic and contents, the webpage is the in-browser display, zoom in/out functionality and language.

File: internal/q4_spectaprod [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* [Automations for platform-test-suite](https://trello.com/c/hBqUETkb) - **Completed** This bounty automated manual operations required to work with the platform test suite.
```
[the remark this answers is not in the document]
Comment (spectaprod) [reply internal/q4_spectaprod]: it turned out to only be the individual task (the .env automation). We had both anticipated more, but no.

File: internal/q4_spectaprod [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* [Goal/Challenge Platform](https://trello.com/c/8Hy6K7fE)
+* [Dash Incubator Document Repository](https://trello.com/c/PIpozna4)
+* [DashSee](https://trello.com/c/UALbkO5H)
+* [Decentralized Dynamic Application Infrastructure](https://trello.com/c/q16cvr5A)
+
+I am the strategist of a handful of other bounties that for various reasons are in temporarily inactive, though I have been in contact with contributors about what is needed to begin progressing again.
+
+### Budget Request
+
+I am requesting 1000 DASH from the Incubator general budget.  This will cover the budget to fund the projects I'm administering directly, as well as continuing to build a reserve fund for launching/completing large projects in the future.
```
[the remark this answers is not in the document]
Comment (spectaprod) [reply internal/q4_spectaprod]: changed

File: internal/q4_spectaprod [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+My requested budget for the period was 300 Dash, of which 184 was Distributed.
+
+My Q3 proposal aslo included the following bounties on which I didn't make any progress:
+
+* [Atomic Swaps](https://trello.com/c/o9l91FLG) - This was help up because the contributor assigned to QA wasn't able to complete the work. A new contributor has been assigned.
+
+### Quarterly Priorities & Value Propositions
+
+During the upcoming In that time I plan to focus on the above Bounties and additionally (depending on the status of Platform):
```
[the remark this answers is not in the document]
Comment (riongull) [reply internal/q4_spectaprod]: Still here after your last commit

File: internal/2022_q4_spectaprod [hunk 5]
```diff
@@ -36,8 +36,8 @@ During the upcoming In that time I plan to focus on the above Bounties and addit
 * [DashSee](https://trello.com/c/UALbkO5H)
 * [Decentralized Dynamic Application Infrastructure](https://trello.com/c/q16cvr5A)
 
-I am the strategist of a handful of other bounties that for various reasons are in temporarily inactive, though I have been in contact with contributors about what is needed to begin progressing again.
+I am the strategist for a handful of other bounties that, for various reasons, are temporarily inactive. I have been in contact with contributors about what is needed to begin progressing again.
 
 ### Budget Request
 
-I am requesting 750 DASH from the Incubator general budget.  This will cover the budget to fund the projects I'm administering directly, continue continuing to build a reserve fund (currently stands at 116) for launching/completing large projects in the future.
+I am requesting 600 DASH from the Incubator general budget.  This will cover the budget to fund the projects I'm administering directly, continue continuing to build a reserve fund (currently stands at 116) for launching/completing large projects in the future.
```
Comment (riongull) [inline internal/2022_q4_spectaprod:43]: Please update this:
>  600 DASH from the Incubator general budget.
to 
> 200 DASH/month for 3 months.

Dash superblocks pay out monthly so I have to request a monthly amount, repeated for 3 months in our external Incubator proposal. I want internal proposals to mirror that, so request a monthly amount for the quarter.  Oh, and again, you're not requesting from the "Incubator general fund", so just leave that part out.

Also, please fix the "continue continuing" typo.

**Review by riongull — CHANGES_REQUESTED** [review]
Just a couple issues remain before I can merge this.  
- One from above (last review)
- One below that I just made.

**Pull request merged**